### PR TITLE
Albums route

### PR DESCRIPTION
**Pull request by HTran106**

### CODE REVIEW

File: backend/db/migrations/20220517200503-create-song.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ module.exports = {
         type: Sequelize.INTEGER,
         references: {
           model: 'Albums'
-        }
+        },
+        onDelete: 'SET NULL'
```
Comment (WJVincent) [inline backend/db/migrations/20220517200503-create-song.js]: Do you want this to set null or to delete?

Comment (HTran106) [reply backend/db/migrations/20220517200503-create-song.js]: null because song can be a single without an album? that was what i was thinking

Comment (WJVincent) [reply backend/db/migrations/20220517200503-create-song.js]: Sounds good to me

**Review by WJVincent — APPROVED** [review]
Looks good, you'll just need to resolve the merge conflicts

**Pull request merged**